content/300-guides/200-deployment/301-edge/550-deploy-to-deno-deploy.mdx

```diff
@@ -158,85 +155,45 @@ If you use VS Code and see the error `An import path cannot end with a '.ts' ext
+## 5. Enable Accelerate in the Prisma Data Platform
 
-      If the connection to the database is successful, Prisma Data Platform creates your project and redirects your browser to the **Get started** page. On the **Get started** page, you can [optionally link your GitHub repository](/data-platform/classic-projects/platform/projects/create#optional-enable-schema-synchronization) to keep your Prisma schema up-to-date.<br /><br />
-      For a project in Prisma Data Platform, you need a linked Prisma schema if you plan to use [Data Browser](/data-platform/classic-projects/platform/data-browser) and [Query Console](/data-platform/classic-projects/platform/query-console).<br /><br />
-      Deno Deploy requires a GitHub repository and you create that in [Create a repository and push to GitHub](#8-create-a-repository-and-push-to-github).
+To get started with Prisma Accelerate:
 
-      ![Prisma Data Platform - Create project - Paste database connection string, select Data Proxy region, enable static IPs](../../../doc-images/data-platform/projects/create-01-paste-connection-string.png)
+1. Sign up for a free [Prisma Data Platform account](https://console.prisma.io/)
+1. Create a project
+1. Navigate to the project you created
+1. Enable Accelerate
+1. Generate an Accelerate connection string and copy it to your clipboard
 
-1. Create a Data Proxy connection string.
+## 6. Configure Prisma Accelerate in your environment
 
-   1. On the **Get started** screen, click **Create a new connection string**.
-   1. Enter a name for the connection string and click **Create**.
-   1. Copy the `prisma://` connection string to your clipboard.
+With the Accelerate connection string copied, you can replace the existing connection string that you used to create the database schema in your `.env` file.
 
-      ![Prisma Data Platform - Create project - Paste database connection string, select Data Proxy region, enable static IPs](../../../doc-images/data-platform/projects/create-03-copy-data-proxy-connection-string.png)
-
-   1. Save the connection string in a password manager or another safe location.
-   1. Click **Skip and continue to Data Platform** to open the Data Browser for your project.<br /><br />
-      When you test your application later, you come back to the Data Browser to review the records that your application creates.
-
-### What's next
-
-Next in this guide, you use the connection string in your local `.env` file to connect to your database through the Data Proxy.
-
-## 6. Configure the Data Proxy connection string in your environment
-
-With the Data Proxy connection string copied, you can replace the direct PostgreSQL connection string that you used to create the database schema in your `.env` file.
-
-Prisma Client for the Data Proxy does not read `.env` files by default, so you must also install `dotenv-cli` locally.
+Prisma Client for Prisma Accelerate does not read `.env` files by default, so you must also install `dotenv-cli` locally.
```

Comment:
"Prisma Client for Prisma Accelerate" does not exist. You can call it "Prisma Client for Deno" in this case maybe.